src/item.cpp

```diff
@@ -2592,33 +2592,120 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
+                        encumb_data["Legs"] = encumb_data.at( "leg_r" );
+                        encumb_data.erase( encumb_data.find( "leg_r" ) );
+                    }
+                    if( covers( bodypart_id( "hand_l" ) ) && covers( bodypart_id( "hand_r" ) )
+                        && encumb_data.at( "hand_l" ).first == encumb_data.at( "hand_r" ).first ) {
+                        encumb_data.erase( encumb_data.find( "hand_l" ) );
+                        encumb_data["Hands"] = encumb_data.at( "hand_r" );
+                        encumb_data.erase( encumb_data.find( "hand_r" ) );
+                    }
+                }
+
+                for( auto &encumb : encumb_data ) {
+                    if( t->sided ) {
+                        const std::string &covering_id = std::get<0>( encumb );
+                        if( !covers( bodypart_id( "arm_l" ) ) && covering_id == "arm_l" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "arm_r" ) ) && covering_id == "arm_r" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "hand_r" ) ) && covering_id == "hand_r" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "hand_l" ) ) && covering_id == "hand_l" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "leg_l" ) ) && covering_id == "leg_l" )  {
+                            continue;
+                        } else if( !covers( bodypart_id( "leg_r" ) ) && covering_id == "leg_r" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "foot_r" ) ) && covering_id == "foot_r" ) {
+                            continue;
+                        } else if( !covers( bodypart_id( "foot_l" ) ) && covering_id == "foot_l" ) {
+                            continue;
+                        }
+                    }
+                    const std::string &bodypart_name = encumb.first;
+                    const std::tuple<int, int, int> &coverage_data = encumb.second.first;
+
+                    // Already human-readable as manually hardcoded just above this
+                    // so just translate string rather than grabbing translation
+                    if( bodypart_name == "Feet" || bodypart_name == "Arms" || bodypart_name == "Legs" ||
+                        bodypart_name == "Hands" ) {
+                        info.push_back( iteminfo( "ARMOR", _( bodypart_name ) + ':' + space, "",
+                                                  iteminfo::no_newline | iteminfo::lower_is_better,
+                                                  std::get<0>( coverage_data ) ) );
+                    } else {
+                        info.push_back( iteminfo( "ARMOR",
+                                                  _( encumb.second.second.translated() )  + ':' + space, "",
```

Comment:
```suggestion
                                                  encumb.second.second.translated()  + ':' + space, "",
```
`translated` returns a translated string. No need to translate it again.